portal/src/pages/theme.vue

```diff
@@ -141,7 +141,8 @@ export default {
         .dispatch('getTheme', themeId)
         .then(theme => {
           let themeCategory = this.$store.getters.getCategoryById(
-            theme.external_id
+            theme.external_id,
+            '8a422e6d-1c35-4961-a50a-51fcff7cc20c'
```

Comment:
is dit overal hetzelfde? Kunnen we daar niet beter ergens een constant voor maken?

Reply:
Ja, dat is inderdaad wel duidelijker.